frontend/src/Commons/Forms/InputModules/CalculateNutrients/CalculateNutrients.tsx

```diff
@@ -396,7 +397,7 @@ const CalculateNutrientsComponent: FC<InputModuleProps> = ({
                       label="Date (optional)"
                       name="applDate"
                       id="applDate"
-                      type="text"
+                      type="date"
```

Comment:
Nice, didn't know this was an option, good to learn :smile:

Reply:
Yeah! It was something I've saw when googling.